### PR TITLE
Update migration script

### PR DESCRIPTION
**Pull request by mpgxvii**
- Update migration script to make MP client and secret a configurable argument (was previously getting `401 bad credentials` but didn't realise it was because of the secret)

### CODE REVIEW

**Review by baixiac — APPROVED** [review]
LGTM.

File: scripts/clients.py [hunk 1]
```diff
@@ -35,8 +35,8 @@ def get_all_notifications_from_hsqldb(self) -> list:
 
 class MpClient:
 
-    def __init__(self, mp_url, user="admin", password="admin", client_id="ManagementPortalapp",
-                 client_secret="") -> None:
+    def __init__(self, mp_url, user="admin", password="admin", client_secret="", 
+                 client_id="ManagementPortalapp") -> None:
```
Comment (yatharthranjan) [inline scripts/clients.py:39]: You don't need to change the order here if you are using named arguments when calling the function.

File: scripts/migrate-xmpp-to-appserver.py [hunk 2]
```diff
@@ -88,7 +91,8 @@ def main():
                                  args.hsqldb_properties,
                                  args.hsqldb_driver_path,
                                  )
-    mp_client = MpClient(args.managementportal_base_url, args.managementportal_user, args.managementportal_password)
+    mp_client = MpClient(args.managementportal_base_url, args.managementportal_user,
```
Comment (yatharthranjan) [inline scripts/migrate-xmpp-to-appserver.py:94]: as above, you can use named arguments like - 

```
mp_client = MpClient(
                                   args.managementportal_base_url,
                                   user = args.managementportal_user, 
                                   password = args.managementportal_password,
                                   client_secret =  args.managementportal_secret
                                  )
```

File: scripts/migrate-xmpp-to-appserver.py [hunk 3]
```diff
@@ -66,6 +66,9 @@ def main():
     parser.add_argument("--managementportal-password", type=str, default="admin",
                         help="The password for the user to authorise with using password grant code.")
 
+    parser.add_argument("--managementportal-secret", type=str, default="",
+                        help="The client secret for ManagementPortalapp client for authorising with password grant code.")
+
```
Comment (yatharthranjan) [inline scripts/migrate-xmpp-to-appserver.py:71]: Can you rename this to `managementportal-client-secret` and also add config for `managementportal-client-id` with default `ManagementPortalapp `?

**Review by yatharthranjan — COMMENTED** [review]
Thanks for the changes, please find some comments below

**Review by yatharthranjan — APPROVED** [review]
LGTM. Thanks for the changes.

The CI is failing as the radar-spring library is still not migrated to maven central. I will try to do it when i have some free bandwidth.

**Pull request merged**